### ISSUE TITLE
give list of images instead of directory to "flow_from_directory" in keras image data generator.

### ISSUE THREAD

**Issue by imohamadhoseins** [tensorflow/tensorflow#46883]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 2.2.0
- Are you willing to contribute it (Yes/No): be an honor to.

**Describe the feature and the current behavior/state.** now, we just can pass directories of train, test and validation to the flow_from_directory for building the generator in keras. i want to give a list of image pathes with their label (like two lists) to build the generator.

**Will this change the current api? How?** no.

**Who will benefit with this feature?** me

**Any Other info.** thanks

**Comment by AdityaKane2001** [comment]
This provision already exists, though not directly in keras. Tensorflow offers tf.data module (docs [here](https://www.tensorflow.org/api_docs/python/tf/data)) ,which consists of tf.data.Dataset. It can be used in any way that suits you. It can be optimized by usage of tf.function decorator. Building such input pipelines is demonstrated in [tutorial](https://www.tensorflow.org/guide/data) ,[tutorial](https://www.tensorflow.org/guide/data_performance). Please check out these links. Hope this helps.

**Comment by amahendrakar** [comment]
@imohamadhoseins,
Please take a look at @AdityaKane2001's comment and let us know if it helps. Thanks!
